docs/recipes/intregration-with-rxangular.md

```diff
@@ -0,0 +1,76 @@
+# Integration with RxAngular
+There are use cases when is desired to have a transient component state has it's lifetime bound to the component lifecycle.
+[RxAngular](https://github.com/rx-angular/rx-angular) provides a solution for this problem that is fully reactive and with focus on runtime performance and template rendering.
+
+In order to leverage this library and all the power of a global state management is possible to integrate RxAngular with NGXS.
+First let's bind a value provinient from NGXS state to RxAngular component state: 
+
+```ts
+interface HeroesComponentState {
+  heroes: Hero[];
+}
+
+const initHeroesComponentState: Partial<HeroesComponentState> = {
+  heroes: []
+};
+
+@Component({
+  selector: 'app-heroes',
+  templateUrl: './heroes.component.html',
+  styleUrls: ['./heroes.component.css'],
+  providers: [RxState] // <--- This binds RxState to the lifetime of the component
+})
+export class HeroesComponent {
+  @Select(getHeroes) SateHeroes$: Obersavable<Hero[]>; // <--- normal @Select use from NGXS
```

Comment:
@Carniatto Super minor but I think there're two typos in this line :)

SateHeroes => S**t**ateHeroes
Obersavable => Ob**ser**vable

Reply:
StateHeroes => stateHeroes :smile:

Reply:
```suggestion
  @Select(getHeroes) stateHeroes$: Observable<Hero[]>; // <--- normal @Select use from NGXS
```